googleapis/google-cloud-node
# migrate code from googleapis/nodejs-org-policy

- chore: add initial files for launch
- feat!: initial stub of library
- feat: add tests and samples to library
- fix: name
- chore: release 1.0.0
- build: fix package name (#5)
- docs: update repo url and linkinator config (#8)
- chore(deps): update dependency sinon to v10 (#10)
- chore: migrate to owl bot (#11)
- chore(deps): update dependency @types/sinon to v10 (#14)
- chore(deps): update dependency ts-loader to v9 (#18)
- fix(deps): require google-gax v2.12.0 (#22)
- chore: new owl bot post processor docker image (#24)
- fix: use require() to load JSON protos (#25)
- chore: update gapic-generator-typescript to v1.3.2 (#26)
- chore: release 1.0.1 (#23)
- chore(deps): update dependency sinon to v11 (#29)
- fix: GoogleAdsError missing using generator version after 1.3.0 (#30)
- chore: release 1.0.2 (#31)
- fix: make request optional in all cases (#37)
- chore: release 1.0.3 (#38)
- fix(deps): google-gax v2.17.0 with mTLS (#41)
- chore: release 1.0.4 (#43)
- fix(deps): google-gax v2.17.1 (#44)
- chore: release 1.0.5 (#45)
- fix: Updating WORKSPACE files to use the newest version of the Typescript generator. (#46)
- chore: release 1.0.6 (#47)
- chore(nodejs): update client ref docs link in metadata (#52)
- fix(deps): google-gax v2.24.1 (#54)
- chore: release 1.0.7 (#55)
- feat: turns on self-signed JWT feature flag (#56)
- chore: release 1.1.0 (#57)
- chore: disable renovate dependency dashboard (#1194) (#59)
- fix(build): switch primary branch to main (#60)
- chore: release 1.1.1 (#61)
- build(node): update deps used during postprocessing (#1243) (#68)
- chore: fix the wrong post processor image (#70)
- chore(deps): update dependency @types/node to v16 (#69)
- chore(cloud-rad): delete api-extractor config (#72)
- docs(samples): add example tags to generated samples (#73)
- chore(deps): update dependency sinon to v12 (#71)
- build: add generated samples to .eslintignore (#74)
- docs(node): support "stable"/"preview" release level (#1312) (#77)
- chore: add api_shortname and library_type to repo metadata (#76)
- docs(badges): tweak badge to use new preview/stable language (#1314) (#79)
- test(nodejs): remove 15 add 16 (#1322) (#81)
- build: update copyright year to 2022 (#83)
- chore: update v2.12.0 gapic-generator-typescript (#85)
- chore(deps): update dependency sinon to v13 (#87)
- docs(nodejs): version support policy edits (#1346) (#89)
- fix: deprecate AlternativePolicySpec (#90)
- chore(main): release 1.1.2 (#91)
- docs(samples): include metadata file, add exclusions for samples to handwritten libraries (#92)
- chore: update v2.14.2 gapic-generator-typescript (#96)
- build(node): update client library version in samples metadata (#1356) (#103)
- chore(deps): update dependency @types/mocha to v9 (#104)
- chore(deps): update dependency sinon to v14 (#107)
- build!: update library to use Node 12 (#108)
- chore(main): release 2.0.0 (#110)
- chore(deps): update dependency jsdoc-region-tag to v2 (#114)
- chore(deps): update dependency jsdoc-fresh to v2 (#113)
- fix(docs): describe fallback rest option (#115)
- chore(main): release 2.0.1 (#116)
- chore(deps): update dependency linkinator to v4 (#120)
- fix: better support for fallback mode (#121)
- fix: change import long to require (#122)
- chore(main): release 2.0.2 (#123)
- fix: do not import the whole google-gax from proto JS (#1553) (#125)
- fix: allow passing gax instance to client constructor (#126)
- fix: preserve default values in x-goog-request-params header (#128)
- test: use fully qualified request type name in tests (#130)
- fix(deps): use google-gax v3.5.2 (#134)
- chore(deps): update dependency @types/node to v18 (#132)
- fix: regenerated protos JS and TS definitions (#137)
- chore(deps): update dependency jsdoc to v4 (#136)
- chore(main): release 2.0.3 (#127)
- build: add release-please config, fix owlbot-config

Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:

Fixes #<issue_number_goes_here> 🦕
